corehq/apps/userreports/dbaccessors.py

```diff
@@ -48,7 +48,7 @@ def get_datasources_for_domain(domain, referenced_doc_type=None, include_static=
             static_ds = [ds for ds in static_ds if ds.referenced_doc_type == referenced_doc_type]
         datasources.extend(sorted(static_ds, key=lambda config: config.display_name))
 
-    return sorted(datasources, key=lambda config: config.display_name)
+    return datasources
```

Comment:
Isn't this the one you want to leave, and take out the `sorted`s on lines 35 and 49?

Alternatively, you could change line 49 to 
```python
    datasources.extend(static_ds).sort(key=lambda config: config.display_name)
```

Reply:
From my other comment:

>  in most places we have normal data sources ordered before static ones

By this I mean that when we order data sources, we generally do so like:

* display name 1 - (data source)
* display name 3 - (data source)
* display name 2 - (static data source)

So if we were to sort the entire list it would mix those based on the display name. Here I intentionally sort the data sources, and then append a sorted list of the static data sources to keep the same behavior

Reply:
Ah, OK. That makes perfect sense.